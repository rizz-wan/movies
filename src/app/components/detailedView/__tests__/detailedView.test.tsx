import { DetailedView } from "..";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("DetailedView tests", () => {
  describe("Snapshot tests", () => {
    it("Should render detailed view page and match snapshot", () => {
      const result = shallow(<DetailedView />);
      expect(result).toBeTruthy();
      expect(result).toMatchSnapshot();
    });
  });
});
