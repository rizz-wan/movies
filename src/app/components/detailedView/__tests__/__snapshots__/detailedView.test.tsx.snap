// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailedView tests Snapshot tests Should render detailed view page and match snapshot 1`] = `ShallowWrapper {}`;
